_rules/80f0bf.md

```diff
@@ -0,0 +1,112 @@
+---
+name: No auto-play audio
+test_type: composite
+
+description: |  
+  This rule checks that auto-play audio does not last for more than 3 seconds, or it has a control mechanism.
+
+success_criterion: 
+- 1.4.2 # Audio Control
+
+atomic_rules:
+- 95e043
+- aaa1bf
+
+authors:
+- Anne Thyme Nørregaard
+- Bryn Anderson
+---
+
+## Applicability
+
+This rule applies to any HTML `audio` or `video` element with a `src` file content `duation` of more than 3 seconds, is set to `autoplay` and or `loop`, and has audio output that is not `paused` or `muted`.
+
+## Expectation
+
+For each test target, the outcome of at least one of the following rules is passed:
+- Auto-play audio does not last more than 3 seconds
+- Auto-play audio has a control mechanism
+ 
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility Support
+
+There are no major accessibility support issues known for this rule.
+
+## Test Cases
+
+### Passed
+
+### Passed example 1
+
+The `audio` element has a control mechanism.
+
+``` html
+  <audio src="../test-assets/moon-audio/moon-speech.mp3" autoplay="true" controls></audio>
+```
+
+### Passed example 2
+
+The `video` element does not play for longer than 3 seconds.
+
+``` html
+ <video autoplay="true" muted="true">
+  <source src="../test-assets/rabbit-video/video.mp4#t=8,10" type="video/mp4" />
+  <source src="../test-assets/rabbit-video/video.webm#t=8,10" type="video/webm" />
+</video>
+```
+
+### Failed
+
+### Failed example 1
+
+The `audio` element autoplays and does not have a control mechanism.
+
+``` html
+  <audio src="../test-assets/moon-audio/moon-speech.mp3" autoplay="true" controls></audio>
+```
+
+### Failed example 2
+
+The `video` element audio autoplays for longer than 3 seconds.
+
+``` html
+ <video autoplay="true" muted="true">
+  <source src="../test-assets/rabbit-video/video.mp4" type="video/mp4" />
+  <source src="../test-assets/rabbit-video/video.webm" type="video/webm" />
+</video>
+```
+
+### Inapplicable
+
+### Inapplicable example 1
+
+The `video` element is `muted`.
+
+``` html
+ <video autoplay="true" muted="true">
+  <source src="../test-assets/rabbit-video/video.mp4" type="video/mp4" />
+  <source src="../test-assets/rabbit-video/video.webm" type="video/webm" />
+</video>
+```
+
+### Inapplicable example 2
+
+The `video` element has no audio output.
+
+``` html
+ <video autoplay="true" muted="true">
+  <source src="../test-assets/rabbit-video/video-with-incorrect-voiceover.mp4" type="video/mp4" />
+  <source src="../test-assets/rabbit-video/video-with-incorrect-voiceover.webm" type="video/webm" />
+</video>
+```
+
+### Inapplicable example 3
+
+The `audio` element does no `autoplay` attribute.
```

Comment:
does no `autoplay` > does not have an `autoplay` 
And remove the `autoplay` from the example :-)